cocos/spine/attach-util.ts

```diff
@@ -35,75 +35,60 @@ const tempMat4 = new Mat4();
  * @class sp.AttachUtil
  */
 export class AttachUtil {
-    protected _inited = false;
-    protected _skeleton: spine.Skeleton | null = null;
-    protected _skeletonNode: Node|null = null;
-    protected _skeletonComp: Skeleton|null = null;
+    protected isInitialized = false;
+    protected skeletonBones: spine.Bone[] | FrameBoneInfo[] | undefined;
+    protected socketNodes: Map<number, Node> | undefined;
+    private keysToDelete: number[] = [];
```

Comment:
Why remove `_` prefix for protected and private variables?

Reply:
Fixed this issue.